### PR TITLE
[SPARK-30384][WEBUI]Needs to improve the Column name and Add tooltips for the Fair Scheduler Pool Table

### PR DESCRIPTION
**Pull request by 07ARB**
### What changes were proposed in this pull request?
Needs to improve the Column name and tooltips for the Fair Scheduler Pool Table.

### Why are the changes needed?
Need to correct SchedulingMode  column name to  -> 'Scheduling Mode' and tooltips need to add for Minimum Share, Pool Weight and Scheduling Mode (require meaning full Tool tips for the end user to understand.)

### Does this PR introduce any user-facing change?
YES
Screenshot 2020-01-03 at 10 10 47 AM

### How was this patch tested?
Manual Testing.

### CODE REVIEW

**Comment by 07ARB** [comment]
@srowen ,@cloud-fan and @HyukjinKwon please review this PR.

**Comment by HyukjinKwon** [comment]
cc @gengliangwang

File: core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala [hunk 1]
```diff
@@ -34,11 +34,21 @@ private[ui] class PoolTable(pools: Map[Schedulable, PoolData], parent: StagesTab
     <table class="table table-bordered table-striped table-condensed sortable table-fixed">
       <thead>
         <th>Pool Name</th>
-        <th>Minimum Share</th>
-        <th>Pool Weight</th>
+        <th>
+          <span data-toggle="tooltip" data-placement="top" title="Pools can be set a minimum share
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: These are a bit too long for tooltips. They are meant to elaborate, very briefly, the immediate meaning of the column. More like "Pool's minimum share of CPU cores"

File: core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala [hunk 2]
```diff
@@ -34,11 +34,21 @@ private[ui] class PoolTable(pools: Map[Schedulable, PoolData], parent: StagesTab
...
+           of CPU cores to allocate">Minimum Share</span>
+        </th>
+        <th>
+          <span title="This controls the pool's share of the cluster relative to other pools"
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: share of what? "Pool's share of cluster resources relative to others"

File: core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala [hunk 3]
```diff
@@ -34,11 +34,21 @@ private[ui] class PoolTable(pools: Map[Schedulable, PoolData], parent: StagesTab
...
+           data-toggle="tooltip"  data-placement="top">Pool Weight</span>
+        </th>
         <th>Active Stages</th>
         <th>Running Tasks</th>
-        <th>SchedulingMode</th>
+        <th>
+          <span title="Mode to control whether jobs within the pool queue up behind each other
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: Not sure if we can meaningfully elaborate the mode here, maybe leave it.
Remember that any real docs would exist in the documentation, not here

**Review by srowen — CHANGES_REQUESTED** [review]

File: core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala (hunk 1 above)
Comment (07ARB) [reply core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: "Pool's minimum share of CPU cores" - This is fine, i will updated the code

File: core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala (hunk 2 above)
Comment (07ARB) [reply core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: ok i will update the tooltips to this "Pool's share of cluster resources relative to others"

File: core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala (hunk 3 above)
Comment (07ARB) [reply core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: ok i will remove the tooltips for mode.

Comment (07ARB) [reply core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: @srowen,  this document i referred to construct tooptips https://spark.apache.org/docs/3.0.0-preview/job-scheduling.html , please check "Configuring Pool Properties" section.

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Jenkins test this please

**Comment by 07ARB** [comment]
@zhengruifeng , please merge this PR.

**Comment by 07ARB** [comment]
@HyukjinKwon and @zhengruifeng , can you please review this MR.

File: core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala [hunk 4]
```diff
@@ -34,11 +34,17 @@ private[ui] class PoolTable(pools: Map[Schedulable, PoolData], parent: StagesTab
...
         <th>Pool Name</th>
-        <th>Minimum Share</th>
-        <th>Pool Weight</th>
+        <th>
+          <span data-toggle="tooltip" data-placement="top" title="Pool's minimum share of CPU
+           cores">Minimum Share</span>
+        </th>
+        <th>
+          <span data-toggle="tooltip" data-placement="top" title="Pool's share of cluster resources
+           relative to others pools">Pool Weight</span>
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: others pools -> others

Comment (07ARB) [reply core/src/main/scala/org/apache/spark/ui/jobs/PoolTable.scala]: done

**Review by HyukjinKwon — APPROVED** [review]

**Comment by srowen** [comment]
Jenkins retest this please

**Comment by gengliangwang** [comment]
@07ARB Could you update the screenshot in the PR description?

**Review by gengliangwang — APPROVED** [review]
LGTM except one comment

**Comment by 07ARB** [comment]
done

**Comment by gengliangwang** [comment]
Thanks, merging to master

**Pull request closed without merging**